models/user.py

```diff
@@ -304,6 +310,9 @@ def last_schedule_paste(self):
     def get_user_courses(self):
```

Comment:
Wouldn't it make rename this method to ```user_courses``` and put the method under an ```@property``` decorator? The method doesn't take in any parameter except ```self```.